classes/user/PKPUserDAO.inc.php

```diff
@@ -243,7 +244,7 @@ function getAllReviewers($contextId) {
 	 * @param $reviewRoundId int Also filter users assigned to this round of the given submission
 	 * @return array Users
 	 */
-	function getFilteredReviewers($contextId, $stageId, $doneMin = null, $doneMax = null, $avgMin = null, $avgMax = null, $lastMin = null, $lastMax = null, $activeMin = null, $activeMax = null, $interests = array(), $submissionId = null, $reviewRoundId = null) {
+	function getFilteredReviewers($contextId, $stageId, $name = null, $doneEnabled = null, $doneMin = null, $doneMax = null, $avgEnabled = null, $avgMin = null, $avgMax = null, $lastEnabled = null, $lastMin = null, $lastMax = null, $activeEnabled = null, $activeMin = null, $activeMax = null, $interests = array(), $submissionId = null, $reviewRoundId = null) {
```

Comment:
I agree it's an obnoxious line of code.

As I understand it, the `lastMin` and `lastMax` values are passed along in the form submission, even if `lastEnabled` is checked. I can see two approaches:

1. We strip out the values in JavaScript before they're posted to the server, if the field is disabled. This will require a lot of specific code in `ClientFormHandler.js` which we probably don't want.

2. Right around [here](https://github.com/pkp/pkp-lib/pull/1064/files#diff-66ad5c92e421039f092ebca5dc27a26aR163) we reset the min/max values to `null` if the enabled value is false.

The second seems better, yea?